### ISSUE TITLE
[QUESTION/BUG] Clicking header panel does not bring it to front. !! Refers to jsPanel v3.x !!

### ISSUE THREAD

**Issue by temuri416** [Flyer53/jsPanel#70]
Hello,

1. Go to http://jspanel.de/api/#gettingstarted
2. Execute example 1 twice
3. Click header of the panel that was opened first

You will see that it will remain behind the panel that was opened last. Clicking panel body will bring it to front.

Shouldn't the same happen if you click panel header?

Thanks!

**Comment by Flyer53** [comment]
Hi again,
basically you're right. I remember having an issue with that because the titlebar is a handle for the dragit interaction. I need to check whether that's still the case after the latest updates ...
Meanwhile I'll see whether I have a workaround and post it here.

Btw: That's working as you expect with jsPanel http://alpha.jspanel.de/

**Comment by Flyer53** [comment]
Just realized that you filed this under jsPanel version 2.x where it works as you expect but you referenced a link to v3.x
So I guess it's about v3.x where you can do the following:
```javascript
$.jsPanel({
    callback: function(panel){
        $(panel.header.title).on('mousedown', function(){
            panel.front();
        });
    }
});
```
Looks like I can change that with the next update to work with the titlebar as well.

**Comment by Flyer53** [comment]
**!! Refers to jsPanel v3.x !!**
I have an updated version 3.9.2.
Not an official release yet, but you can download from [http://jspanel.de/downloads/jsPanel-3.9.2.zip](http://jspanel.de/downloads/jsPanel-3.9.2.zip)
Should improve options dragit/resizeit including clicking the header to front the panel (panels are also fronted now as soon as you start dragging or resizing them). Also fixes an issuw with option.onwindowresize.
Please give it a try ... Thanks :)

**Comment by Flyer53** [comment]
**!! Refers to jsPanel v3.x !!**
Fixed with release of v3.9.2
